Gemfile

```diff
@@ -101,7 +101,7 @@ gem 'memcached'
 gem 'jbuilder'
 
 # Curb for curl bindings
-gem 'curb'
+gem 'curb', '0.8.5'
```

Comment:
why updating to this specific version?
on my dev box i have weird problems getting curb 0.8.5 to install.
if you need this, then i can get ggtrackerstack up and running, i expect it'll work fine there.